test/metabase/api/setting_test.clj

```diff
@@ -10,12 +10,15 @@
 (use-fixtures :once (fixtures/initialize :db))
 
 ;; ## Helper Fns
-(defn- fetch-test-settings  []
+(defn- fetch-test-settings
+  "Feting all settings"
```

Comment:
nit: `Fetching`

but also maybe the docstring should clarify that it only fetches test settings, with `test-setting` in their name